server/doc/swdesign/plantuml/seq_update_state.plantuml

```diff
@@ -19,9 +19,12 @@ activate agent1
 agent1 ->> tonic_server--++: UpdateState
 tonic_server ->> server--++: UpdateState
 server -> server_state++: update state
-server_state -> server_state++--: create FromServer message\nfrom comparison of new\nand old CurrentState
+server_state -> server_state ++--: Compare new with\nold CurrentState
+server_state -> server_state ++--: Validate the new state
+server_state -> server_state ++--: Update DeleteGraph with\nDeleteConditions\nof new workloads
 alt State valid
-server_state -->> server--: FromServer message
+server_state -->> server--: Added and deleted workloads
```

Comment:
The activation is missing here. Its also missing earlier in the diagram before the changes in this PR. 
We should update the diagram to get everything consistent.

Reply:
Thanks, sure. I will do that.

Reply:
Yes you are right, but the activates beforehand are missing. Participants cannot make a call without being active and must be activated when called.

Reply:
The lifetime appears in the preview of VSCode but in the exported diagram. After a check, plantuml has a bug, because the version in the preview of VSCode is from 2022 and the one used inside the devcontainer is from 2020. I will open a separate PR with the latest version of plantuml inside the devcontainer and to add a setting so that the preview in VSCode uses the jar file inside the container to create the preview diagram.

Reply:
Can we update the diagram here directly? We don't need to wait for the other PR, right?